package.json

```diff
@@ -8,6 +8,12 @@
   "module": "dist/downshift.esm.js",
   "typings": "typings/index.d.ts",
   "scripts": {
+    "start": "nps",
```

Comment:
why is `nps` being added as part of this PR to add flow definitions? For example, why would someone execute `npm start validate` instead of `npm run validate`?